monitor/getShortEventStats.js

```diff
@@ -1,19 +1,35 @@
 require('dotenv').config()
 const eventsInfo = require('./utils/events')
+const { BRIDGE_MODES } = require('../commons')
 
 async function main(bridgeMode) {
   const { foreignDeposits, homeDeposits, homeWithdrawals, foreignWithdrawals } = await eventsInfo(bridgeMode)
 
-  return {
-    depositsDiff: homeDeposits.length - foreignDeposits.length,
-    withdrawalDiff: homeWithdrawals.length - foreignWithdrawals.length,
-    home: {
-      deposits: homeDeposits.length,
-      withdrawals: homeWithdrawals.length
-    },
-    foreign: {
-      deposits: foreignDeposits.length,
-      withdrawals: foreignWithdrawals.length
+  if (bridgeMode === BRIDGE_MODES.ARBITRARY_MESSAGE) {
+    return {
+      deliveryDiff: homeDeposits.length - foreignDeposits.length,
```

Comment:
for AMB it is necessary to change `checkWorkers.js` as so it outputs:
```
{"home":{"toForeign":1513,"fromForeign":1651},"foreign":{"fromHome":1513,"toHome":1655}, "lastChecked":1568748063,"fromHomeToForeignDiff":0,"fromForeignToHomeDiff":-4,"timeDiff":69}
```

Reply:
Updated the output. Please check https://github.com/poanetwork/tokenbridge/pull/199/commits/9c87c239daa852a311e89545701a2b95e01c6e98